### PR TITLE
Add and fix labs doc and examples

### PR DESCRIPTION
**Pull request by enriquebelarte**
/hold

### CODE REVIEW

**Comment by ybettan** [comment]
@enriquebelarte 
When the WIP prefix is set to the PR title, the bot will hold the PR (see `do-not-merge/work-in-progress` label).
Adding `/hold` to it is redundant :)

Just FYI.

File: docs/lab/README.md [hunk 1]
```diff
@@ -50,9 +50,9 @@ These are Kernel and other relevant software versions for EKS AMI Linux from Kub
 
 EKS can also be used with custom node groups based on [Ubuntu](https://cloud-images.ubuntu.com/docs/aws/eks/). In this case we could use Ubuntu 18.04 or Ubuntu 22.04 in our build step at the KMM CRD as a Docker image, as most likely we will find the kernel headers package matching the nodes within the distribution repository:
 
-``
+```console
```
Comment (ybettan) [inline docs/lab/README.md:53]: Nice! I was not familiar with it.

File: docs/lab/README.md [hunk 2]
```diff
@@ -67,13 +67,17 @@ Underlying OS in nodes running at GKE is Container-Optimized OS by default but i
...
 apt update && apt-cache search linux-headers | grep gke
-``
+```
 
 
 If worker nodes are based on the default GKE option which is COS, we will have to download the kernel sources for the same kernel version matching the node before the build step unless we find a kernel header package matching the exact same version as the one running in our nodes which is not the case today in 5.10.109 kernel used in examples. No major distribution delivers this specific kernel headers version in repositories.
 
+We can download kernel sources for GKE from Chromium OS [repository](https://chromium.googlesource.com/chromiumos/third_party/kernel/) as long as COS [Container-Optimized OS](https://cloud.google.com/container-optimized-os/docs/concepts/features-and-benefits) is based on the open source Chromium OS project.
```
Comment (ybettan) [inline docs/lab/README.md:77]:
```suggestion
We can download kernel sources for GKE from Chromium OS [repository](https://chromium.googlesource.com/chromiumos/third_party/kernel/) as long as [Container-Optimized OS](https://cloud.google.com/container-optimized-os/docs/concepts/features-and-benefits) (COS) is based on the open source Chromium OS project.
```

File: docs/lab/gce/debian-driver.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+                  python3 
+                  #bison \
+                  #flex \
+                  #libelf-dev \
+                  #gnupg \
+                  #wget \
+                  #git \
+                  #make \
+                  #bc 
+                  #gcc
```
Comment (ybettan) [inline docs/lab/gce/debian-driver.yaml:39]: Do we need those comments?

File: docs/lab/gce/debian-driver.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  name: kmm-ci-deb-a
+spec:
+  moduleLoader:
+    container:
+      modprobe:
+        moduleName: kmm-ci-deb-a 
+
+      kernelMappings:
+        - literal: 5.10.127 
+          containerImage: quay.io/ebelarte/kmmo:gke-kmm-deb-1
```
Comment (ybettan) [inline docs/lab/gce/debian-driver.yaml:13]: We should probably use a generic image here like `quay.io/org/image:gke-kmm-deb-1` or something similar.

File: docs/lab/gce/debian-driver.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+    
+              FROM debian:stable-slim 
+              ARG KERNEL_VERSION
+              RUN apt-get update && apt-get install kmod
+              COPY --from=builder /src/kernel/kmm-kmod/kmm_ci_a.ko /opt/lib/modules/${KERNEL_VERSION}/
+              COPY --from=builder /src/kernel/kmm-kmod/kmm_ci_b.ko /opt/lib/modules/${KERNEL_VERSION}/
+              RUN depmod -b /opt
+  imageRepoSecret: 
+    name: quay-eb
```
Comment (ybettan) [inline docs/lab/gce/debian-driver.yaml:62]: Can we use a generic secret name? also adding `<>` will help the user understand it needs to modify it, for example `<my-secret>`

File: docs/labs/README.md [hunk 6]
```diff
@@ -64,18 +64,25 @@ In Standard GKE clusters Control Plane (master) is administered by Google so lab
 
 Underlying OS in nodes running at GKE is Container-Optimized OS by default but it could be Ubuntu also.
 
-As stated in previous platform, in the case of Ubuntu Linux as node OS we should use Ubuntu Linux as the docker image for the build step in KMM CRD as it will almost certainly contain the kernel headers package matching the node kernel version. 
+As stated in previous platform, in the case of Ubuntu Linux as node OS we should use Ubuntu Linux as the docker image for the build step in KMM CRD as it will certainly contain the kernel headers package matching the node kernel version. As of today, Ubuntu 20.04 focal LTS includes packages matching kernel version running in Ubuntu with containerd nodes at GKE. This may change as newer versions of Ubuntu will be used on GKE nodes.
```
Comment (qbarrand) [inline docs/labs/README.md:67]: How is it certain that the latest Ubuntu container image will contain kernel headers for the version GKE is running?

File: docs/labs/README.md [hunk 7]
```diff
@@ -64,18 +64,25 @@ In Standard GKE clusters Control Plane (master) is administered by Google so lab
...
 
 We could get the whole list of available linux-headers packages for GKE just by doing this:
-``
+```console
 apt update && apt-cache search linux-headers | grep gke
-``
+```
+Use of Ubuntu based nodes with containerd in GKE is the straight, preferred and easier option if you are willing to use KMM.
```
Comment (qbarrand) [inline docs/labs/README.md:73]: I would introduce the paragraph that follows and emphasize that it's not recommended:
```suggestion
The default option, Google's Container-optimized OS, comes with security settings that make it difficult to load out-of-tree kernel modules.
Using Ubuntu nodes with containerd in GKE is the straight, preferred and easier option if you are willing to use KMM.
```

File: docs/labs/README.md (hunk 6 above)
Comment (enriquebelarte) [reply docs/labs/README.md:67]: Not the latest, but the version of Ubuntu matching the node.

**Pull request merged**